crates/iceberg/src/spec/table_properties.rs

```diff
@@ -49,6 +53,8 @@ pub struct TableProperties {
     pub write_format_default: String,
     /// The target file size for files.
     pub write_target_file_size_bytes: usize,
+    /// Compression codec for metadata files (JSON)
+    pub metadata_compression_codec: String,
```

Comment:
This should be sth like `Option<String>`, `none` should be `None` in memory.

Reply:
done.